README.md

```diff
@@ -361,6 +361,82 @@ secrets:
       name: xwiki-db-root-password
 ```
 
+## Configuring clustering
+
+Read about [setting communication channels](https://www.xwiki.org/xwiki/bin/view/Documentation/AdminGuide/Clustering/).
+
+#### Docker Compose example
+
+```yaml
+
+version: '2'
+networks:
+  bridge:
+    driver: bridge
+services:
+  web:
```

Comment:
Idea: use `web1` since there's `web2` below.

Reply:
okay I would update that